resource-service/README.md

```diff
@@ -1,10 +1,14 @@
-# New Configuration Service
+# Resource Service :: The New Configuration Service
 
-The *resource-service* is a Keptn core component and used to manage resources for Keptn project-related entities, 
-i.e., project, stage, and service. The entity model is shown below. To store the resources with version control, a git 
-repository is used that is mounted as emptyDir volume.  Besides, this service has functionality to upload the git repository 
+The *resource-service* is a Keptn core component used to manage resources for Keptn project-related entities,
+i.e., project, stage, and service. The entity model is shown below. To store the resources with version control, a git

```

Comment:
Do you mean i.e. here or e.g.?  In other words, are these the specific entities for which can manage resources (which is what i.e. means) or that these are examples of entities?

Reply:
These are all specific entities that the resource-service handles and not examples